### PR TITLE
Doc: add getting-started tutorial

### PR DESCRIPTION
**Pull request by tang-mm**
## Description

Added a basic tutorial for first-time end users, covering:
- installing the CLI via Snap
- configure the default server
- creating a first job
- submit job
- check job status
- check job output

(This tutorial has some overlap with the how-to guides in #143 , with fewer options and more explanation for new users.) 

## Resolved issues

Resolves [CERTTF-222](https://warthogs.atlassian.net/browse/CERTTF-222)

## Documentation

Preview version on Readthedocs:

https://canonical-testflinger.readthedocs-hosted.com/en/doc-preview/tutorial

## Tests

To check the documentation on a local sphinx site:
```
$ cd testflinger/docs
$ make install
$ make run     # Serve docs on http://127.0.0.1:8000/reference/test-phases/
```

[CERTTF-222]: https://warthogs.atlassian.net/browse/CERTTF-222?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/tutorial/index.rst [hunk 1]
```diff
@@ -1,20 +1,228 @@
...
+
+The most convenient way to install the CLI is through the *Snap Store*. Open a terminal and run the following command:
+
+.. code-block:: shell
+  
+  $ sudo snap install testflinger-cli
+
+Once the installation is finished, you can execute the ``testflinger-cli`` command to check the installation. For example, you can run with the ``--help`` or ``-h`` option to display the CLI help page, which is useful anytime when you need a reference about how to use the tool:
+
+.. code-block:: shell
```
Comment (plars) [inline docs/tutorial/index.rst:35]: Can we remove the syntax highlighting from this section? It's highlighting some strange words in the preview build.

File: docs/tutorial/index.rst [hunk 2]
```diff
@@ -1,20 +1,228 @@
...
+
+  $ testflinger-cli list-queues
+
+.. note::
+  
+  If you want to temporarily use another server, add ``--server`` argument and the server URI in the command.
+
+If the connection is successful, a list of job queues is returned with their queue names and short descriptions:
+
+.. code-block:: shell
```
Comment (plars) [inline docs/tutorial/index.rst:113]: Same here with the syntax highlighting. "for" gets highlighted below.

**Review by plars — CHANGES_REQUESTED** [review]
Those preview links are super-helpful, thanks for including those!
This looks good other than some strange syntax highlighting. I would guess it could either be turned off for those code blocks or changed to some other type of highlighting.

**Review by plars — APPROVED** [review]
It looks like the earlier section for testflinger-cli looks fine for me now since you removed "bash", but the later one still had weird highlighting.
image

I found that if I use "text" instead, then it renders fine for me now. I hope you don't mind, I just pushed the change, since it was trivial. And I'm +1 on it now. But I'll let you make the final call :)

**Review by kevinyehk — APPROVED** [review]
LGTM +1

**Pull request merged**